components/image-builder-mk3/workspace-image-layer/gitpod-layer/debian/gitpod/.bashrc-prepend

```diff
@@ -1,58 +0,0 @@
-
-
-###############################################################################
-##########  Gitpod - prepend
-###############################################################################
-
-# Prompt color and bash_completion
-export PS1='\[\e]0;\u \w\a\]${debian_chroot:+($debian_chroot)}\[\033[01;32m\]\u\[\033[00m\] \[\033[01;34m\]\w \$ \[\033[00m\]'
-#source /etc/bash_completion
-
-# editor config - should be removed when registry facade is default
-if [ -z "$EDITOR" ]; then
-    export EDITOR="gp open -w"
-fi
-if [ -z "$VISUAL" ]; then
-    export VISUAL="$EDITOR"
-fi
-if [ -z "$GIT_EDITOR" ]; then
-    export GIT_EDITOR="$EDITOR"
-fi
-
-# Workaround Java pre v10 by explicitly setting "-Xmx" for all Hotspot/openJDK VMs
-if [ -n "$GITPOD_MEMORY" ]; then
-    export JAVA_TOOL_OPTIONS="-Xmx${GITPOD_MEMORY}m";
-fi
-
-# Completion for gp command
-. <(gp completion)
-# ide cli config - should be removed when registry facade is default
-if [ ! -d "/ide/bin/" ]; then
-    alias open='gp open'
-    alias code='gp open'
-fi
-
-export GEM_HOME=/workspace/.rvm
-export GEM_PATH=$GEM_HOME:$GEM_PATH
-export PATH=/workspace/.rvm/bin:$PATH
-
-export PIPENV_VENV_IN_PROJECT=true
-export PIP_USER=yes
-export PYTHONUSERBASE=/workspace/.pip-modules
-export PATH=$PYTHONUSERBASE/bin:$PATH
-unset PIP_TARGET
-unset PYTHONPATH
-
-# Set CARGO_HOME to reside in workspace if:
-#  - it's RUNTIME (/workspace present)
-if [ -d /workspace ]; then
-    export CARGO_HOME=/workspace/.cargo
-    export PATH=$CARGO_HOME/bin:$PATH
-fi
-
-export BROWSER="${BROWSER:=gp-preview}"
-
-###############################################################################
-##########  Gitpod - prepend
-###############################################################################
-
```

Comment:
~~Burn it, burn it all with fire! 🔥🔥🔥~~

TODO:
- [x] `PS1` prompt gets re-defined anyway so line had no effect
- [ ] Test that making a new commit from the console is still okay (`EDITOR` vars)
- [ ] Test that `JAVA_TOOL_OPTIONS` is still defined so that the JVM won't eat up all Gitpod's precious RAM
- [ ] Test `gp` and its auto-completion
- [ ] Test `open` and `code` aliases (nice to haves / but not mandatory)
- [x] `GEM_HOME=/workspace/.rvm` gets overwritten by nearly all Ruby projects anyway
- [x] We want a less crazy-custom Python setup so fine by me
- [ ] Test `cargo install` persistence across restarts
- [x] I've never liked the `BROWSER` automatic behavior (i.e. "automatically open IDE preview when some specific web servers start") (also did that even work in Code?)

I guess most of these changes are fine, because we want fewer non-standard customizations:
- except for `gp` and `JAVA_TOOL_OPTIONS`, which look dangerous and really should be tested
- also, big question: is Full Workspace Backup enabled everywhere now? (If yes, it's fine to remove all `/workspace` hacks -- if not, this PR breaks Cargo, Python and maybe Ruby package persistence)